api/src/main/resources/sql/application.sql

```diff
@@ -132,14 +132,16 @@ CREATE TABLE nrmn.survey_aud (
     direction_mod boolean,
     inside_marine_park varchar(255),
     inside_marine_park_mod boolean,
-    latitude float8,
+    latitude float4,
```

Comment:
These should match (be generated from) from entity definitions in the java code.   I'll fix those.